modules/ROOT/pages/dataweave-formats-xml.adoc

```diff
@@ -35,15 +35,12 @@ _Introduced in DataWeave 2.5.0. Supported by Mule 4.5.0 and later._
 
 DataWeave can read and write doctype declarations in XML files.
 
-To read a doctype declarations, set the xref:dataweave-system-properties.adoc[system property] `com.mulesoft.dw.xml_reader.parseDtd`.
-
-During the reading phase, DataWeave parses the doctype declarations and stores the content as metadata of the root element. The `DocType` value is stored in the `docType` variable. You can use the metadata selector (`^`) to extract the value of that variable with an expression like this one:
+During the reading phase, DataWeave parses the doctype declarations and stores the content as metadata of the root element. The `DocType` value is stored in the `docType` variable. You can use the xref:dataweave-selectors.adoc[metadata selector] (`^`) to extract the value of that variable with an expression like this one:
 
 * `payload.^docType`
 +
 See examples <<example12>>, <<example13>> and <<example14>>.
 
-You can use the xref:dataweave-selectors.adoc[metadata selector] to access it.
```

Comment:
https://docs.mulesoft.com/dataweave/latest/dataweave-selectors[metadata selector] 

if possible to put in the source. Need to check into the options.